src/scene/materials/standard-material-options-builder.js

```diff
@@ -39,8 +39,8 @@ StandardMaterialOptionsBuilder.prototype.updateRef = function (options, device,
     options.hasTangents = objDefs && stdMat.normalMap && ((objDefs & SHADERDEF_TANGENTS) !== 0);
     this._updateLightOptions(options, stdMat, objDefs, sortedLights, staticLightList);
     this._updateUVOptions(options, stdMat, objDefs, false);
-    options.clearCoat = stdMat.clearCoat;
-    options.clearCoatGlossiness = stdMat.clearCoatGlossiness;
+    options.clearCoat = !!stdMat.clearCoat;
+    options.clearCoatGlossiness = !!stdMat.clearCoatGlossiness;
```

Comment:
I wonder if those should be moved to _updateMaterialOptions - that's where the other material options are.

Reply:
I just move the `clearCoat` and `clearCoatGlossiness` to `_updateMaterialOptions`.